sandbox/descriptions.html.example

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
If we did then changes made to them would get included in PRs, which would kind of defeat the point of the sandbox directory. The docs/examples directory is where we can have examples that work out of the box, but those should also probably be included in dist and use the dist files.